Sources/VitalHealthKit/HealthKit/HealthKitReads.swift

```diff
@@ -295,12 +295,14 @@ func handleProfile(
   
   let sex = try healthKitStore.biologicalSex().biologicalSex
   let biologicalSex = ProfilePatch.BiologicalSex(healthKitSex: sex)
-  
-  var components = try healthKitStore.dateOfBirthComponents()
-  components.timeZone = TimeZone(secondsFromGMT: 0)
-  
-  let dateOfBirth = components.date!
-  
+
+  var components = try healthKitStore.patched_dateOfBirthComponents()
+  components?.timeZone = TimeZone(secondsFromGMT: 0)
+  // HealthKit promises that the returned components respect Grogorian calendar.
+  components?.calendar = Calendar(identifier: .gregorian)
```

Comment:
you can use `vitalCalendar`.

Reply:
It is not suited for purpose. `vitalCalendar` is derived from the device current calendar, which does not have to be Gregorian.

Reply:
TimeZone has been set (prior to this PR) to UTC+0 explicitly such that `DateComponents.date` always produces a timestamp at the UTC day 00:00.

Reply:
Standardised everything on `vitalCalendar`, which is now explicitly set as Gregorian